libraries/ui-ios/icons/ShareIcon.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon from '@shopgate/pwa-common/components/Icon';
+
+/**
+ * The filter icon component. SVG is grabbed from config
+ * @param {Object} props The icon component properties.
+ * @returns {JSX}
+ */
+const ShareIcon = (props) => {
+  const content = `<path d="${props.svg}"/>`;
+  return (
+    <div>
+      <Icon content={content} {...props} />
```

Comment:
IMHO there is no reason to have two exact same ShareIcons in two ios/material libraries that render nothing by it's own.
Please change it so it's independent. One should only import the icon and then it should render by it's own. There should be no dependency.

I am aware of the AC that says the extension should be configurable in this matter, but the ui-shared library should not be aware of this. Some suggestions how to support this are posted in another PR: https://github.com/shopgate/pdp-native-share/pull/1/files